src/components/active-preset/active-preset.js

```diff
@@ -16,6 +16,8 @@ const mapStateToProps = state => ({
     shell: state.shell,
     engine: {
       adBreak: state.engine.adBreak,
+      adIsLinear: state.engine.adIsLinear,
+      adContentType: state.engine.adContentType,
```

Comment:
I would change ads preset to be a component (like playback for example) and use these states there